packages/instantsearch.js/src/components/Template/Template.tsx

```diff
@@ -10,6 +10,28 @@ import type { BindEventForHits, SendEventForHits } from '../../lib/utils';
 import type { Templates } from '../../types';
 import type { JSX } from 'preact';
 
+class RawHtml extends Component<{ content: string }> {
+  ref = createRef();
+  nodes: Element[] = [];
+
+  componentDidMount() {
+    const fragment = new DocumentFragment();
+    const root = document.createElement('div');
+    root.innerHTML = this.props.content;
+    this.nodes = [...root.children];
+    fragment.append(...this.nodes);
```

Comment:
appendNode is IE11 compatible if I remember it correctly, not append

Reply:
Ah yes it's `appendChild`, I'll need to use `forEach` too

Would be nice to settle on IE11 for once, as SFCC did already